health/health.go

```diff
@@ -0,0 +1,54 @@
+package health
+
+import (
+	"fmt"
+
+	"github.com/gofiber/fiber/v2"
+	"go.uber.org/fx"
+)
+
+type status struct {
+	Healthy bool
+	Reason  *string
```

Comment:
We might want to add a sync.Mutex to allow concurrent invocations of the status setter functions